jit/src/main/java/org/dcsa/conformance/standards/jit/action/SupplyScenarioParametersAction.java

```diff
@@ -104,7 +41,19 @@ public boolean isInputRequired() {
 
   @Override
   public void handlePartyInput(JsonNode partyInput) {
+    log.info("SupplyScenarioParametersAction.handlePartyInput({})", partyInput.toPrettyString());
     super.handlePartyInput(partyInput);
     suppliedScenarioParameters = SuppliedScenarioParameters.fromJson(partyInput.get("input"));
+
+    dsp =
+      new DynamicScenarioParameters(
```

Comment:
`DynamicScenarioParameters` has historically been used for parameters intercepted by exchanges and not "Pre-declared scenario parameters". The code here uses `DynamicScenarioParameters` for both cases, which makes it a bit more confusing for me (since it is the first time I see them mixed).

If this is a new paradigme, then ignore this comment and I will catch up.

Reply:
Interesting finding. I wasn't aware of any paradigm shift myself. I also build it up, by parameters intercepted by the exchanges, but in this method, I start it, by adding SSP values, so I can leave the SSP for what it is. 
Anyways, I'll check if this should be modified, to make it easier or more consistent.